components/dashboard/src/Menu.tsx

```diff
@@ -182,6 +184,17 @@ export default function Menu() {
         })();
     }, [user, teams, team, project]);
 
+    useEffect(() => {
+        (async () => {
+            const isBlockedRepositoriesUIEnabled = await getExperimentsClient().getValueAsync(
+                "isblockedrepositoriesuienabled",
+                false,
+                {},
```

Comment:
FYI: It's super convenient to just pass in `user` here, which allows us to enable this feature flag more selectively (e.g., for all Gitpodders).

Reply:
Thanks, done in 73eedce